neuralprophet/utils.py

```diff
@@ -490,7 +490,7 @@ def print_epoch_metrics(metrics, val_metrics=None, e=0):
     return metrics_string
 
 
-def fcst_df_to_last_forecast(fcst, n_last=1):
+def fcst_df_to_last_forecast(fcst, quantiles, n_last=1):
     """Converts from line-per-lag to line-per-forecast.
```

Comment:
please expand on the args in a docstring

Reply:
done! Pls see PR #764 as this PR will be closed